src/Oriflame.RedisMessaging.ReliableDelivery/Oriflame.RedisMessaging.ReliableDelivery.csproj

```diff
@@ -1,24 +1,35 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
-  <Import Project="../../props/version.props" />
-  <Import Project="../../props/common.props" />
-
-  <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <PackageTags>oriflame redis publish subscribe reliable guaranteed delivery failover pubsub messaging</PackageTags>
-    <ProjectGuid>{BE5D5D0E-CD40-43A2-92EB-7D0F2BE8222A}</ProjectGuid>
-    <PackageDescription>Extends Redis publish/subscribe so that potentially lost messages can be recovered</PackageDescription>
-  </PropertyGroup>
-
-  <PropertyGroup>
-    <CodeAnalysisRuleSet>Ori.Common.Clustering.Events.Redis.ruleset</CodeAnalysisRuleSet>
-    <LangVersion>7.2</LangVersion>
-  </PropertyGroup>
-
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.2.0" />
-    <PackageReference Include="StackExchange.Redis" Version="2.0.601" />
-  </ItemGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="../../props/version.props" />

```

Comment:
I know, this is not part of this changes (it was there before) but as soon as you would introduce Directory.Build.props (?) this can be part of it?